FFCDemoPaymentService.Tests/MessagingTests/MessageServiceTests.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.ComponentModel;
+using System.IO;
+using NUnit.Framework;
+using Moq;
+using FFCDemoPaymentService;
+using FFCDemoPaymentService.Messaging;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading;
+using System.Threading.Tasks;
+
+
+namespace FFCDemoPaymentService.UnitTests.MessagingTests
+{
+
+    public class MessageServiceTests
+    {        
+        MessageConfig messageConfig;
+        MessageService messageService;
+
+        [SetUp]
+        public void Setup()
+        {
+            messageConfig = new MessageConfig();
+            messageConfig.Host = "123";
+            messageConfig.Port = 111;
+            messageConfig.UseSsl = true;
+            messageConfig.PaymentQueue = "Test_Queue";
+            messageConfig.PaymentUserName = "Test_User";
+            messageConfig.PaymentPassword = "Test_Password";
+            var iConnection = new Mock<IConnection>();
+            var serviceScopeFactory = new Mock<IServiceScopeFactory>();
+            messageService = new MessageService ( iConnection.Object, messageConfig, serviceScopeFactory.Object );
+        }
+
+        [Test]
+        public void ConfigHost()
```

Comment:
I wouldn't bother with these tests.  They're not actually testing any logic within the application.  They're just testing that the values set in the setup still exist when the test runs.  I'd remove them.

Reply:
Removed as suggested